### PR TITLE
Add offline update and beta release install instructions

### PR DESCRIPTION
**Pull request by mcm001**

### CODE REVIEW

**Comment by amquake** [comment]
Should this mention the contribution guide local build instructions as well for installing pre-release versions?

File: source/docs/getting-started/installation/updating.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+In the example release above, we see:
+
+- Image files for flashing directly to supported coprocessors.
+
+  - Raspberry Pi 3/4/5/CM4: follow our :ref:`Raspberry Pi flashing instructions<docs/getting-started/installation/sw_install/raspberry-pi:Raspberry Pi Installation>`.
+  - For LimeLight devices: follow our :ref:`LimeLight flashing instructions<docs/getting-started/installation/sw_install/limelight:Imaging>`.
+  - For Orange Pi 5 devices: follow our :ref:`Orange Pi flashing instructions<docs/getting-started/installation/sw_install/orange-pi:Orange Pi Installation>`.
+
+- JAR files for the suite of supported operating systems for use with Offline Update. In general:
```
Comment (mdurrani808) [inline source/docs/getting-started/installation/updating.rst:17]: what operating systems does offline update not work on?

**Review by mdurrani808 — DISMISSED** [review]

**Comment by mcm001** [comment]
> Should this mention the contribution guide local build instructions as well for installing pre-release versions?

Yeah we should I think

File: source/docs/getting-started/installation/updating.rst (hunk 1 above)
Comment (mcm001) [reply source/docs/getting-started/installation/updating.rst:17]: Windows/Mac, sort of. I think we expect Photon to have been installed as a service

**Pull request merged**